app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -211,25 +226,50 @@ private void showFeed() {
 
         txtvDescription.setText(description);
 
-        if (!TextUtils.isEmpty(feed.getAuthor())) {
-            txtvAuthor.setText(feed.getAuthor());
-            txtvAuthorHeader.setText(feed.getAuthor());
```

Comment:
The author in the header should still be displayed. Please only remove the author from the actual page, not from the blurred header.